src/DDTrace/Integrations/Mysqli/MysqliIntegration.php

```diff
@@ -122,6 +161,47 @@ public static function load()
             return $this;
         });
 
+        // bool mysqli_stmt_get_result ( mysqli_stmt $stmt )
+        dd_trace('mysqli', 'real_connect', function ($stmt) {
+            $args = func_get_args();
+            $tracer = GlobalTracer::get();
+            if ($tracer->limited()) {
+                return dd_trace_forward_call();
+            }
+
+            $scope = MysqliIntegration::initScope('mysqli.real_connect', 'mysqli.real_connect');
+            /** @var \DDTrace\Span $span */
+            $span = $scope->getSpan();
+
+            // PHP 5.4 compatible try-catch-finally
+            $thrown = null;
+            $result = null;
+            try {
+                $result = dd_trace_forward_call();
+                //Mysqli::storeConnectionParams($this, $args);
```

Comment:
Do we need to keep this or uncomment it? :)

Reply:
nope ... copy and paste :)